src/track/bpm.h

```diff
@@ -45,6 +45,10 @@ class Bpm final {
         return isValidValue(m_value);
     }
 
+    bool isNull() const {
```

Comment:
This is not testing for null, so at the very least it should be 

```suggestion
    bool isMinValue() const {
```

However we have other places in the code that simply test for bpm == 0.0 (see https://github.com/mixxxdj/mixxx/blob/main/src/engine/controls/bpmcontrol.cpp#L1107), so either we need to update those locations with the new function, or, my preference, just test for == 0.0.

Reply:
but I think this comment is obviated by the one below